base/combinatorics.jl

```diff
@@ -2,38 +2,40 @@
 
 # Factorials
 
-const _fact_table64 = Vector{Int64}(undef, 20)
-_fact_table64[1] = 1
-for n in 2:20
-    _fact_table64[n] = _fact_table64[n-1] * n
-end
+const _shifted_fact_table64 = [1; cumprod(1:Int64(20))]  # tables shifted by 1 so factorial(0) can be looked up; maybe OffsetArray better?
```

Comment:
Comments in code are not the right place to ask questions. Please remove the question, and also the comments about old code.

Also, let's keep the same names as before.